src/envelope.rs

```diff
@@ -72,53 +56,61 @@ impl HandlerSpan {
         r
     }
 
-    fn disabled() -> HandlerSpan {
+    fn none() -> Span {
         #[cfg(feature = "instrumentation")]
-        let span = HandlerSpan(tracing::Span::none());
+        let span = Span(tracing::Span::none());
 
         #[cfg(not(feature = "instrumentation"))]
-        let span = HandlerSpan();
+        let span = Span();
```

Comment:
It would make the review easier to not have this rename in here, or at least in a commit at the very front of the PR :)